hack/common.sh

```diff
@@ -862,4 +865,109 @@ function os::build::find-binary() {
   local path=$( (ls -t $(os::build::get-bin-output-path "${os_root}")/${bin}) 2>/dev/null || true | head -1 )
   echo "$path"
 }
-readonly -f os::build::find-binary
\ No newline at end of file
+readonly -f os::build::find-binary
+
+# os::build::environment::create creates a docker container with the default variables.
+# arguments are passed directly to the container, OS_BUILD_ENV_GOLANG, OS_BUILD_ENV_IMAGE,
+# and OS_RELEASE_DOCKER_ARGS can be used to customize the container. The docker socket
+# is mounted by default and the output of the command is the container id.
+function os::build::environment::create() {
+  set -o errexit
```

Comment:
Without that it would fail to warn me.

On Wed, Jun 15, 2016 at 2:54 PM, Steve Kuznetsov <notifications@github.com>
wrote:

> In hack/common.sh
> <https://github.com/openshift/origin/pull/9278#discussion_r67223630>:
>
> > \ No newline at end of file
> > +readonly -f os::build::find-binary
> > +
> > +# os::build::environment::create creates a docker container with the default variables.
> > +# arguments are passed directly to the container, OS_BUILD_ENV_GOLANG, OS_BUILD_ENV_IMAGE,
> > +# and OS_RELEASE_DOCKER_ARGS can be used to customize the container. The docker socket
> > +# is mounted by default and the output of the command is the container id.
> > +function os::build::environment::create() {
> > +  set -o errexit
>
> this is surprising
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9278/files/8437cd96443c7aee0b866981c9a57910926e0646#r67223630>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p8tLhbDD6m6lS-eqTEZPjCl39kn_ks5qMEp9gaJpZM4IzMty>
> .
>